src/components/ProjectVersion/index.jsx

```diff
@@ -1,19 +1,176 @@
-import React from "react";
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
 import styled from "styled-components";
 
+import Modal from "../shared/Modal";
+import Loading from "../shared/Loading";
 import Title from "../shared/Title";
 import Select from "../shared/Select";
 import BottomNavigator from "../shared/BottomNavigator";
 
+import { getPages } from "../../services/pages";
+
+import useProjectVersionStore from "../../../store/projectVersion";
+import usePageListStore from "../../../store/projectPage";
+import usePageStatusStore from "../../../store/projectInit";
+
 function ProjectVersion() {
+  const [isLoaded, setIsLoaded] = useState(true);
+  const [beforeDate, setBeforeDate] = useState("");
+  const [afterDate, setAfterDate] = useState("");
+  const [beforeVersion, setBeforeVersion] = useState("");
+  const [afterVersion, setAfterVersion] = useState("");
+  const navigate = useNavigate();
+
+  const { allDates, byDates } = useProjectVersionStore(state => state);
+  const { status, setStatus, clearProject } = usePageStatusStore();
+  const { setPages } = usePageListStore();
+
+  const onChangeBeforeDate = ev => {
+    const date = ev.target.value;
+
+    setBeforeDate(date);
+  };
+
+  const onChangeBeforeVersion = ev => {
+    const version = ev.target.value;
+
+    setBeforeVersion(version);
+  };
+
+  const onChangeAfterDate = ev => {
+    const date = ev.target.value;
+
+    setAfterDate(date);
+  };
+
+  const onChangeAfterVersion = ev => {
+    const version = ev.target.value;
+
+    setAfterVersion(version);
+  };
+
+  const dateFilter = () => {
```

Comment:
P1: 함수명은 동사형으로 변경해주시면 감사하겠습니다 👍 

```suggestion
  const filterLaterDates = () => {
```

Reply:
함수명 변경했습니다.
dateFilter => filterNextDates